Makefile

```diff
@@ -523,10 +524,12 @@ kind-image: ## Build cilium-dev docker image and import it into kind.
 	@echo "  DEPLOY image to kind ($(LOCAL_AGENT_IMAGE))"
 	$(QUIET)$(CONTAINER_ENGINE) push $(LOCAL_AGENT_IMAGE)
 	$(QUIET)kind load docker-image $(LOCAL_AGENT_IMAGE)
+ifeq ($(SKIP_OPERATER),"false")
```

Comment:
```suggestion
ifeq ($(SKIP_OPERATOR),"false")
```

Reply:
Woops. I checked that this disabled the operator build, which was effective.. but forgot to check that the default still enables the operator build. I'll fix this up.